cocos/platform/CCDataManager.h

```diff
@@ -47,7 +48,7 @@ class CC_DLL DataManager
     static void onSceneLoaderEnd();
     static void onShaderLoaderBegin();
     static void onShaderLoaderEnd();
-    static void setOptimise(const char* thing, const char* value);
+    static void setOptimise(std::string thing, std::string value);
```

Comment:
std::string -> const std::string&

Reply:
@xianyinchen   You forget to add `const` in your new commit.

Use `const std::string& thing` instead of `std::string&`.

Reply:
Oh, i didn't notice it.